content/topics/good-health-and-wellbeing/stories/Beibarys-Nyussupov-story.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Comparing child mortality and child malnutrition in High and Low sustainable development areas in Bolivia
+author: Beibarys Nyussupov
+date: 2023-04-29
+---
+
+## **Varibles Used :**
+| Variable Name | Description |
+| :-- | :-: |
+| <b>imds</b>|Municipal Sustainable Development Index|
+|<b>sdg2_2_cmc</b>|Chronic malnutrition in children (< 5 years), 2016 (%)|
+|<b>sdg3_2_mrc</b>|Children mortality rate in (< 5 years), 2016 (per. 1,000 live births)|
+|<b>index_sdg1</b>|Rate of contribution to the no poverty goal|
+|<b>sdg1_4_abs</b>|Access to the 3 basic services, 2012 (% of households)|
+
+
+{{< paragraph class="large" >}}
+A researchers tried to understand the difference in health area in cities with varying rates of sustainable development. Municipal Sustainable development index (imds) was used to identify 5 cities with highest and 5 cities with lowest levels of sustainable development in Bolivia. The bar plot was visualized and it was determined that cities with high sustainable development had scores from 70 to 80, while cities with low sustainable development had scores from 30 to 40, showing a significant difference in development.
+{{</ paragraph >}}
+
+
+{{< image class="medium" src="https://raw.githubusercontent.com/DSSD-Madison/Nagoya/Story/Beibarys/content/topics/good-health-and-wellbeing/stories/images/Image1.png" alt="Comparing Low and High sustainable Development index" >}}
```

Comment:
I used new shortcodes that you have added for my story. Furthermore, I made my graphic visualizations better and referenced my graphic images to the path local to the project, rather than to my branch in Github. I think story is ready to be merged to main.